ghost-schema.js

```diff
@@ -1,133 +1,157 @@
+    slug: String!
+    id: ID! 
+    uuid: String!
+    title: String!
+    html: String!
+    comment_id: String!
+    feature_image: String
+    featured: Boolean!
+    visibility: String!
+    created_at: String!
+    updated_at: String!
+    published_at: String!
+    custom_excerpt: String
+    codeinjection_head: String
+    codeinjection_foot: String
+    codeinjection_styles: String
+    custom_template: String
+    canonical_url: String
+    send_email_when_published: Boolean
+    tags: [GhostTag] @link(from: "tags.slug" by: "slug")
+    authors: [GhostAuthor]! @link(from: "authors.slug" by: "slug")
+    primary_author: GhostAuthor! @link(from: "primary_author.slug" by: "slug")
+    primary_tag: GhostTag @link(from: "primary_tag.slug" by: "slug")
+    url: String!
+    excerpt: String!
+    reading_time: Int!
+    og_image: String
+    og_title: String
+    og_description: String
+    twitter_image: String
+    twitter_title: String
+    twitter_description: String
+    meta_title: String
+    meta_description: String
+    email_subject: String
+}
 
-const settings = {
-    title: 'Ghost',
-    description: 'The professional publishing platform',
-    logo: 'https://static.ghost.org/v1.0.0/images/ghost-logo.svg',
-    icon: 'https://static.ghost.org/favicon.ico',
-    cover_image: 'https://static.ghost.org/v1.0.0/images/blog-cover.jpg',
-    facebook: 'ghost',
-    twitter: '@ghost',
-    lang: 'en',
-    timezone: 'Etc/UTC',
-    codeinjection_head: '<script>>some script</script><style></style>',
-    codeinjection_foot: '<style>.some-class {\n}</style><script></script>',
-    codeinjection_styles: '.some-class {\n}',
-    navigation: [
-        {label: 'Home', url: '/'},
-        {label: 'Tag', url: '/tag/getting-started/'},
-        {label: 'Author', url: '/author/ghost/'},
-        {label: 'Help', url: 'https://help.ghost.org'}
-    ]
-};
+type GhostPage implements Node {
+    slug: String!
+    id: ID! 
+    uuid: String!
+    title: String!
+    html: String!
+    comment_id: String!
+    feature_image: String
+    featured: Boolean!
+    visibility: String!
+    created_at: String!
+    updated_at: String!
+    published_at: String!
+    custom_excerpt: String
+    codeinjection_head: String
+    codeinjection_foot: String
+    codeinjection_styles: String
+    custom_template: String
+    canonical_url: String
+    send_email_when_published: Boolean
+    tags: [GhostTag] @link(from: "tags.slug" by: "slug")
+    authors: [GhostAuthor]! @link(from: "authors.slug" by: "slug")
+    primary_author: GhostAuthor! @link(from: "primary_author.slug" by: "slug")
+    primary_tag: GhostTag @link(from: "primary_tag.slug" by: "slug")
+    url: String!
+    excerpt: String!
+    reading_time: Int!
+    og_image: String
+    og_title: String
+    og_description: String
+    twitter_image: String
+    twitter_title: String
+    twitter_description: String
+    meta_title: String
+    meta_description: String
+    email_subject: String
```

Comment:
```suggestion
    email_subject: String
    mobiledoc: String
    plaintext: String
    page: Boolean
```

This relates to comments about fields from "fake" post/page missing in GraphQL types - I don't know if this is correct - both of all those fields are actually there in GhostAPI response and if those should be nullable or not (so I didn't add `!` in types)